app/routes.py

```diff
@@ -1,2 +1,292 @@
-from flask import Blueprint
+from flask.wrappers import Response
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request
+from datetime import datetime, timezone
+import os
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
```

Comment:
The Slack SDK is fine, but you can also do this with the built-in request object from Python.